crates/bevy_input/src/axis.rs

```diff
@@ -2,8 +2,12 @@ use bevy_utils::HashMap;
 use std::hash::Hash;
 
 #[derive(Debug)]
+/// An axis that stores f32 values in range from `min` to `max`.
+/// By default `min` and `max` equal `-1.0` and `1.0`.
```

Comment:
```suggestion
/// An axis that stores f32 values in range from `min` to `max`.
/// By default `min` and `max` equal `-1.0` and `1.0`.
#[derive(Debug)]
```
This is a better style, as it prevents Attributes from being "hidden" by large Documentation.